### PR TITLE
[IL-326] UI Tests. Shopping Cart Page

### PR DESCRIPTION
**Pull request by ilsinyakov**
1. **Task's name**: [IL-326] UI Tests. Shopping Cart Page
2. **Developer's full name**: Ilya Sinyakov
3. **Changes**:
- **New file**:
UI/pages/cart_page.py
UI/test_shopping_cart.py
- **Changed files**:
UI/configs.py - remove xfail marks from last name positive data (bug is fixed)
UI/pages/locators.py - new locators

### CODE REVIEW

File: UI/test_shopping_cart.py
Comment (TetianaPerinha) [inline UI/test_shopping_cart.py:1]: 1. test_guest_full_cart -  it looks like e2e testing ?  In one test you verify add, update, and remove product from cart.  
So I suggest to rename that test case - test_guest_cart_functionality
What do you think?
2. test_guest_cart_cost - you verify a subtotal of the cart, I think it better to rename it to - 
 test_guest_verify_cart_subtotal
3. test_user_full_cart the same as in 1
4. test_user_cart_cost the same as in 2

Comment (ilsinyakov) [reply UI/test_shopping_cart.py:1]: @TetianaPerinha 
Thanks for review ))

> So I suggest to rename that test case - test_guest_cart_functionality

Done

>  I think it better to rename it to - test_guest_verify_cart_subtotal

Done

> test_user_full_cart the same as in 1
> test_user_cart_cost the same as in 2

Done

+ changed some locators
+ removed xfail marks - bugs is fixed

**Review by TetianaPerinha — APPROVED** [review]
Approve

**Review by TetianaPerinha — APPROVED** [review]
Approve

**Pull request merged**